server/src/main/java/org/opensearch/rest/action/cat/RestShardsAction.java

```diff
@@ -283,6 +283,7 @@ protected Table getTableWithHeader(final RestRequest request) {
 
         table.addCell("path.data", "alias:pd,dataPath;default:false;text-align:right;desc:shard data path");
         table.addCell("path.state", "alias:ps,statsPath;default:false;text-align:right;desc:shard state path");
+        table.addCell("docs.deleted", "alias:dd;default:false;text-align:right;desc:number of deleted docs in shard");
```

Comment:
Any reason not to include "docsDeleted" as an alias? The other _cat APIs with this field seem to have that alias as well.

Reply:
Missed that, added it now.